framework-cloud/framework-openshift/src/main/java/org/kie/cloud/openshift/deployment/DatabaseDeploymentImpl.java

```diff
@@ -0,0 +1,133 @@
+    private String username;
+    private String password;
+    private String namespace;
+    private String database;
+    private URL url;
+    private String serviceName;
+
+    public void setOpenShiftController(OpenShiftController openShiftController) {
+        this.openShiftController = openShiftController;
+    }
+
+    public void setUsername(String username) {
+        this.username = username;
+    }
+
+    public void setPassword(String password) {
+        this.password = password;
+    }
+
+    public void setNamespace(String namespace) {
+        this.namespace = namespace;
+    }
+
+    public void setDatabase(String database) {
+        this.database = database;
+    }
+
+    public void setUrl(URL url) {
+        this.url = url;
+    }
+
+    public OpenShiftController getOpenShiftController() {
+        return openShiftController;
+    }
+
+    @Override
+    public URL getUrl() {
+        return url;
+    }
+
+    @Override
+    public String getUsername() {
+        return username;
+    }
+
+    @Override
+    public String getPassword() {
+        return password;
+    }
+
+    @Override
+    public String getDatabase() {
+        return database;
+    }
+
+    @Override
+    public String getNamespace() {
+        return namespace;
+    }
+
+    public String getServiceName() {
+        return serviceName;
+    }
+
+    public void setServiceName(String applicationName) {
+        this.serviceName = applicationName + "-" + database;
+    }
+
+    @Override
+    public void scale(int instances) {
+        openShiftController.getProject(namespace).getService(getServiceName()).getDeploymentConfig().scalePods(instances);
+    }
+
+    @Override
+    public void waitForScale() {
+        openShiftController.getProject(namespace).getService(getServiceName()).getDeploymentConfig().waitUntilAllPodsAreReady();
+    }
+
+    @Override
+    public List<Instance> getInstances() {
+        String deploymentConfigName = openShiftController.getProject(namespace).getService(getServiceName()).getDeploymentConfig().getName();
+        List<Pod> pods = openShiftController.getClient().pods().inNamespace(namespace).withLabel(OpenShiftResourceConstants.DEPLOYMENT_CONFIG_LABEL, deploymentConfigName).list().getItems();
+
+        List<Instance> databaseInstances = new ArrayList<>();
+        pods.stream().map((pod) -> {
+            DatabaseInstanceImpl databaseInstance = new DatabaseInstanceImpl();
+            databaseInstance.setOpenShiftController(openShiftController);
+            databaseInstance.setNamespace(namespace);
+            databaseInstance.setName(pod.getMetadata().getName());
+            return databaseInstance;
+        }).forEach((databaseInstance) -> {
+            databaseInstances.add(databaseInstance);
+        });
+
+        return databaseInstances;
+    }
+
+    @Override
+    public boolean ready() {
+        throw new UnsupportedOperationException("Not supported yet.");
```

Comment:
yet? Is this going to be implemented soon or never? Are we waiting for something to happen so this can be implemented? Some explanation about why it's not implemented might be useful for others using this..